frame/header-mmr/src/lib.rs

```diff
@@ -0,0 +1,189 @@
+//! # Chain MMR Pallet
+//!
+//! \## Overview
```

Comment:
why add a `\` prefix?

Reply:
It's copy from the sample pallet

Reply:
https://github.com/paritytech/substrate/blob/v2.0.0-alpha.4/frame/example/src/lib.rs#L127

Should used in <pre> code snippets
Fixed.